skyuxconfig.json

```diff
@@ -16,5 +16,10 @@
     "unit": {
       "browserSet": "paranoid"
     }
+  },
```

Comment:
Do we need to also remove `skyuxModules`?

Reply:
Done.